File: examples/lh-server/python/21-structdefs/workflow.py
from littlehorse.workflow import Workflow, WorkflowThread
from tasks import Car


def quickstart_wf() -> Workflow:
    def my_entrypoint(wf: WorkflowThread) -> None:
        input_car = wf.declare_struct("input-car", Car).required()
        wf.execute("describe-car", input_car)

    return Workflow("quickstart", my_entrypoint)

File: examples/lh-server/go/21-structdefs/workflow.go
package main

import "github.com/littlehorse-enterprises/littlehorse/sdk-go/littlehorse"

func QuickstartWorkflow(wf *littlehorse.WorkflowThread) {
	inputCar := wf.DeclareStruct("inputCar", CarStructDefName).Required()
	wf.Execute("describe-car", inputCar)
}
